solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -659,10 +656,11 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
             lockTokens(msg.sender, delta);
         }
```

Comment:
Is there no need to `incNetworkTokensMinted` above this as well (`networkTokenGovernance.mint(address(this), delta - networkBalance);`)?

Reply:
no, since it's not part of the minting for a specific pool, it's minting due to IL/compensation (also discussed it with eyal)